bin/impl/base.sh

```diff
@@ -32,6 +32,11 @@ if [ ! -d $FLUO_HOME ]; then
   echo "FLUO_HOME=$FLUO_HOME does not exist"
   exit 1
 fi
+: ${FLUO_YARN_HOME?"FLUO_YARN_HOME must be set in bash env or conf/webindex-env.sh"}
```

Comment:
The `init` command could be written to support multiple launchers. User could choose which to use with the following usage: `webindex init [--yarn|--kubernetes|--marathon]`